install.sh

```diff
@@ -175,7 +185,8 @@ setup_env() {
     if [ -n "${INSTALL_K3S_TYPE}" ]; then
         SYSTEMD_TYPE="${INSTALL_K3S_TYPE}"
     else
-        if [ "${CMD_K3S}" = "server" ]; then
+        # TODO: simplify to "always notify" when support for < 0.7.0 is dropped
+        if version_ge 'v0.7.0' || [ "${CMD_K3S}" = "server" ]; then
```

Comment:
No quotes, neither " nor ', are needed for "server" in the `[ "${CMD_K3S}" = "server" ]` check.
`[ "${CMD_K3S}" = server ]` does the job.

Reply:
(But it one really want quotes there, it should be single ones since nothing is to be expanded)